source/includes/steps-remove-a-schema-ui.yaml

```
@@ -0,0 +1,57 @@
+title: Navigate to the Collection Schema Screen
+ref: navigate-to-the-collection-schema-screen
+level: 4
+content: |
+  In the {+leftnav+}, click :guilabel:`Schema` beneath :guilabel:`Data
+  Access` to open the schema editor. Realm scans your linked cluster for
+  existing collections and lists them on the left side of the schema
+  editor.
+
+  By default, you see all the collections in your linked Atlas cluster.
+  Any collection that has a schema shows a bolded name in this list, 
+  while a collection with no schema shows a light-gray, italicized name.
+
+  From here, you can remove the schema for a single collection, or 
+  remove the schema for all collections.
+---
+title: Remove the Schema for a Collection
+ref: remove-a-collection-schema
+level: 4
+content: |  
+  1. Hover over the collection name, and a :guilabel:`...` menu appears.
+
+  #. When you click this :guilabel:`...` element, you get two options. One of
+     them is to :guilabel:`Delete Schema from this Collection`. Choose this 
+     option.
+
+  #. A modal pops up asking you to confirm that you want to delete the schema
+     for the collection. Press the :guilabel:`Delete` button.
+
+  #. If you have already terminated {+sync+}, the schema deletes. If you have 
+     not yet :ref:`terminated {+sync+} <terminating-realm-sync>`, 
+     you see a modal letting you know that this is a destructive change, and 
```

```suggestion
     a modal displays, letting you know that this is a destructive change and 
```
"you see" also sounds odd to me, suggesting a more natural wording and removing an unneeded comma

---

Another good catch - thanks! I was originally taking screenshots as I drafted this, but decided not to include them because the UI keeps changing - doesn't make much sense without the screenshots, though. Thanks!